flink-connector-kafka-e2e-tests/flink-end-to-end-tests-common-kafka/src/test/java/org/apache/flink/tests/util/kafka/SQLClientKafkaITCase.java

```diff
@@ -75,22 +76,36 @@ public static Collection<Object[]> data() {
         return Arrays.asList(new Object[][] {{"3.2.3", "universal", "kafka", ".*kafka.jar"}});
     }
 
-    private static Configuration getConfiguration() {
-        // we have to enable checkpoint to trigger flushing for filesystem sink
-        final Configuration flinkConfig = new Configuration();
-        flinkConfig.setString("execution.checkpointing.interval", "5s");
-        return flinkConfig;
-    }
+    private static final Slf4jLogConsumer LOG_CONSUMER = new Slf4jLogConsumer(LOG);
+    public static final String INTER_CONTAINER_KAFKA_ALIAS = "kafka";
+
+    @ClassRule public static final Network NETWORK = Network.newNetwork();
+
+    @ClassRule
+    public static final KafkaContainer KAFKA =
+            new KafkaContainer(DockerImageName.parse(DockerImageVersions.KAFKA))
+                    .withNetwork(NETWORK)
+                    .withNetworkAliases(INTER_CONTAINER_KAFKA_ALIAS)
+                    .withLogConsumer(LOG_CONSUMER);
 
-    @Rule
-    public final FlinkResource flink =
```

Comment:
@zentol I actually propose to get rid of this test. Reason being is that `SQLClientSchemaRegistryITCase` tests SQL Kafka table source and sink already.

This test is just testing E2E the json format -> avro format -> csv format via file sink. However, now that we are using containerized Flink, the file it writes to is now within the container and it is difficult to obtain the file from the taskmanager containers to assert on the file output contents.

We could easily fix this test if we can use LocalStandaloneFlinkResourceFactory (but it is in flink-end-to-end-test-common).

Reply:
Plus I had to create an extraneous module to get the jar of a custom sql function (see flink-sql-toolbox module).

Reply:
Alternatively, I can totally rewrite and simplify the test. (only testing json format -> avro format and asserting Kafka output contents, but this is what `SQLClientSchemaRegistryITCase` is essentially doing).

Reply:
Whoops wrong PR